der/derive/src/lib.rs

```diff
@@ -121,6 +121,12 @@
     unused_qualifications
 )]
 
+macro_rules! abort {
```

Comment:
This got me confused for a minute. I was wondering where `abort` was coming from since you were deleting `proc-macro-error`.